crates/holochain/src/core/sys_validate.rs

```diff
@@ -120,7 +120,7 @@ pub async fn check_spam(_header: &Header) -> SysValidationResult<()> {
 
 /// Check previous header timestamp is before this header
 pub fn check_prev_timestamp(header: &Header, prev_header: &Header) -> SysValidationResult<()> {
-    if header.timestamp() > prev_header.timestamp() {
+    if header.timestamp() >= prev_header.timestamp() {
```

Comment:
did compiling give you a warn error about this or something?

Reply:
I get `CellError(WorkflowError(SourceChainError(InvalidCommit("The previous header\'s timestamp is not before the current header\'s timestamp"))))` most of time without this fix.